### PR TITLE
login buttons style

### PR DESCRIPTION
**Pull request by TinasheMzondiwa**
Closes Issue #159 

Screenshot 2021-02-15 at 13 17 30

### CODE REVIEW

File: features/lessons/src/main/java/com/cryart/sabbathschool/lessons/ui/quarterlies/SSQuarterlyItemViewModel.kt [hunk 1]
```diff
@@ -89,6 +89,7 @@ internal class SSQuarterlyItemViewModel(
         fun loadCover(view: ImageView, coverUrl: String?, primaryColor: Int?) {
             coverUrl?.let { url ->
                 view.load(url) {
+                    allowHardware(false)
```
Comment (TinasheMzondiwa) [inline features/lessons/src/main/java/com/cryart/sabbathschool/lessons/ui/quarterlies/SSQuarterlyItemViewModel.kt]: Fixes crash on api 26 devices

File: features/lessons/src/main/java/com/cryart/sabbathschool/lessons/ui/readings/SSReadingActivity.kt [hunk 2]
```diff
@@ -176,7 +176,7 @@ class SSReadingActivity : SSBaseActivity(), SSReadingViewModel.DataListener, Vie
 
     override fun onDestroy() {
         try {
-            (binding.ssReadingAppBar.ssCollapsingToolbarBackdrop.drawable as BitmapDrawable).bitmap.recycle()
+            (binding.ssReadingAppBar.ssCollapsingToolbarBackdrop.drawable as? BitmapDrawable)?.bitmap?.recycle()
```
Comment (TinasheMzondiwa) [inline features/lessons/src/main/java/com/cryart/sabbathschool/lessons/ui/readings/SSReadingActivity.kt:179]: Fixes crash on api 26 devices

File: features/lessons/src/main/java/com/cryart/sabbathschool/lessons/ui/quarterlies/SSQuarterlyItemViewModel.kt (hunk 1 above)
Comment (merlin-zaraza) [reply features/lessons/src/main/java/com/cryart/sabbathschool/lessons/ui/quarterlies/SSQuarterlyItemViewModel.kt]: It's not fear to disable hardware acceleration for all devices. Maybe just disable crossfade for API 26?

Comment (TinasheMzondiwa) [reply features/lessons/src/main/java/com/cryart/sabbathschool/lessons/ui/quarterlies/SSQuarterlyItemViewModel.kt]: Updated

**Review by merlin-zaraza — APPROVED** [review]

**Pull request merged**